src/libpmemfile/path_resolve.c

```diff
@@ -69,7 +69,7 @@ get_stat(struct resolved_path *result, struct stat *buf)
 		int r = pmemfile_fstatat(result->at.pmem_fda.pool->pool,
 			result->at.pmem_fda.file,
 			result->path,
-			buf,
+			(struct pmemfile_stat *)buf,
```

Comment:
Or just change the get_stat function's signature to accept pmemfile_stat.
The syscall_no_intercept function doesn't care about types anyways.